### ISSUE TITLE
Can't fit GammaDistribution or initialize it with .summarize or .from_summaries

### ISSUE THREAD

**Issue by TrentBrick** [jmschrei/pomegranate#583]
I tried fitting a Gamma Distribution to data that looks exponential just to see how it would fit. No matter what I set the parameters to (I did a grid search from 0-10 for the alphas and betas) it doesnt learn anything. 

I then try to initialize using from_summaries and get this error which I can't find any documentation for how to resolve. 

````
TypeError                                 Traceback (most recent call last)
<ipython-input-57-b30b2e8078fa> in <module>
      2 data = trans.values.reshape(-1,1)
      3 #g = GammaDistribution.summarize(data, weights=None)
----> 4 GammaDistribution.from_summaries(data,weights=None)

TypeError: descriptor 'from_summaries' requires a 'pomegranate.distributions.GammaDistribution.GammaDistribution' object but received a 'numpy.ndarray'
````

**Comment by jmschrei** [comment]
Howdy

Sorry you're having trouble. I know others have had difficulties with GammaDistribution in the past. Have you gotten the latest version of pomegranate?

It looks like you might be using GammaDistribution incorrectly. Neither `summarize` nor `from_summaries` are class methods. This means that you need to initialize objects before using those methods. Alternatively, you need to use the `from_samples` method, as in `GammaDistribution.from_samples(data)`. 

Let me know how that works for you!

**Comment by TrentBrick** [comment]
Hey thanks for the speedy response. 

I installed pomegranate today using conda. And have version '0.10.0'

I initialized the GammaDistribution first now and then ran `from_summaries` and it worked. 

It is a little unintuitive to first have to initalize the GammaDistribution with random values and then reinitialize it again using `from_summaries`

However, it still isn't learning parameters which is weird because it does produce weights? 
`data = trans.values.reshape(-1,1)
g1 = GammaDistribution(2,0.5)
g2 = GammaDistribution(1,0.5)
g1.from_summaries(data)
g2.from_summaries(data)
m = GeneralMixtureModel([g1,g2])
print(m.fit(data, n_jobs=4, stop_threshold=0.001))`

Output: 

`{
    "class" : "GeneralMixtureModel",
    "distributions" : [
        {
            "class" : "Distribution",
            "name" : "GammaDistribution",
            "parameters" : [
                NaN,
                NaN
            ],
            "frozen" : false
        },
        {
            "class" : "Distribution",
            "name" : "GammaDistribution",
            "parameters" : [
                NaN,
                NaN
            ],
            "frozen" : false
        }
    ],
    "weights" : [
        0.16454523816136193,
        0.835454761838638
    ]
}`

-----

Random side note: 
I'm not sure what case you use summarize in (I have only skimmed the docs) but I am getting this error which seems kind of weird, if its summarizing data why would it require a square matrix?:

`g.summarize(data)

pomegranate/distributions/GammaDistribution.pyx in pomegranate.distributions.GammaDistribution.GammaDistribution.summarize()

ValueError: shapes (10888,1) and (10888,1) not aligned: 1 (dim 1) != 10888 (dim 0)`
